cmake/Config.cmake

```diff
@@ -226,3 +226,7 @@ set (BOND_STACK_OPTIONS
 if (BOND_ENABLE_GRPC AND ((CXX_STANDARD LESS 11) OR (MSVC_VERSION LESS 1800)))
     message(FATAL_ERROR "BOND_ENABLE_GRPC is TRUE but compiler specified does not support C++11 standard")
 endif()
+
+if(CMAKE_CXX_STANDARD AND ("${CMAKE_CXX_STANDARD}" GREATER_EQUAL 17))
```

Comment:
For compatibility, Bond builds with with CMake 3.5.1. It doesn't have `GREATER_EQUAL`, which was added in CMake 3.6. If you want to use this, you'll need to write it as a conjunction of an `EQUAL` and a `GREATER`.

Reply:
Ah, that explains why build was failing. Let me go ahead and do comparison with 17 or 20.